docker-compose.yml

```diff
@@ -44,10 +44,15 @@ services:
       EQ_RABBITMQ_HOST_SECONDARY: rabbit
       EQ_SECRETS_FILE: docker-secrets.yml
       EQ_KEYS_FILE: docker-keys.yml
+      EQ_SUBMITTED_RESPONSES_TABLE_NAME: dev-submitted-responses
+      EQ_DYNAMODB_ENDPOINT: http://eq-docker-dynamodb:8000
+      AWS_ACCESS_KEY_ID: dummy
+      AWS_SECRET_ACCESS_KEY: dummy
```

Comment:
We should discourage people from accidentally committing their AWS credentials by having a default and allowing overriding via environment variables (this is done in https://github.com/ONSdigital/eq-compose/blob/master/eq.yml#L91) e.g. `AWS_ACCESS_KEY_ID: "${AWS_ACCESS_KEY_ID:-dummy}"`